web/decap-config/collections/10-anytime-action.yml

```diff
@@ -257,6 +257,41 @@ collections:
                   required: true,
                   hint: Only applies to tab 2 and 3,
                 }
+              - {
+                  label: "Requesting Agency Error Text",
+                  name: "requestingAgencyErrorText",
+                  widget: "string",
+                  required: true,
+                  hint: Only applies to tab 2 and 3,
```

Comment:
pebble - the hint texts need to be reviewed, I don't think the error text appear in tab 3

Reply:
fixed